### DISCUSSION TITLE
enhancement request

### DISCUSSION THREAD

**Discussion by bernieEV** [electricitymaps/electricitymaps-contrib#3235] [Ideas]
at the national level there are the neighbor countries with import/export info. (table on the left side)
I would like to see a summary line below to show the total number of the normalized values  (sum of all neighbors) either import or export

thanks 
BernieEV (located in beautiful Austria)
P.S. congratulation to this real good infomation about electricity generation and consumption in Europe

**Comment by madsnedergaard** [comment]
Hi @bernieEV, thanks for the suggestion!
I've moved your idea to discussions - unfortunately we don't have the capacity to prioritise this internally right now, but if anyone else wants to do it they're very welcome to pick it up :)
